### PR TITLE
about-section-style-changes

### PR DESCRIPTION
**Pull request by Hydroelectric29**
WHAT DOES THIS PR DO? 
1. Centered all sub-headers in the about section. 
2. Decreased the font size for text. 
3. Removed unnecessary space between sub-header and text. 

How to test this PR 
1. Run: gh pr checkout 30 && npm install  && npm run dev 
2. Scroll down to bottom About section. 
3. Review style changes. 
4. Thank you very much :)

### CODE REVIEW

File: src/App.jsx [hunk 1]
```diff
@@ -37,22 +37,22 @@ const App = () => {
...
-    //   .then(newMetrics => setMetrics(newMetrics))
-    //   .catch(err => console.log(err));
+    fetch('http://localhost:3000/metrics')
+      .then(res => res.json())
+      .then(newMetrics => setMetrics(newMetrics))
+      .catch(err => console.log(err));
   }, []);
 
-  if (isSplashShowing) return (<Splash />);
+  // if (isSplashShowing) return (<Splash />);
```
Comment (dlande000) [inline src/App.jsx]: whoops, we forgot to recomment the Splash component back in. Could you please update line 55 to show our splash page again?

File: src/components/about/About.jsx [hunk 2]
```diff
@@ -1,38 +1,46 @@
...
                 Kubiquity is a Kubernetes error logging and tracking tool developed and launched in 2021. Designed for production-level applications using Kubernetes to orchestrate containers, Kubiquity provides metrics on all pods' memory use to help developers prevent OOM kill errors; additionally, Kubiquity logs Kubernetes events to highlight any errors and warnings that originate within monitored clusters.
             </p>
         </div>
         <div className="about-section">
-            <div className="about-subheader">
+            <div
+                className="about-subheader"
+                style={ABOUT_SECTION_STYLE}
+            >
+
```
Comment (dlande000) [inline src/components/about/About.jsx]: Looks like we accidentally included an extra line. Could you please delete line 29?

**Review by dlande000 — APPROVED** [review]

**Comment by dlande000** [comment]
Because of changes that have been made to the app (in PR 29), and because of our different indentations, assuming PR 29 goes in first, this is going to have some merge conflicts for us to resolved. Once the PR is fully reviewed, we'll go over those conflicts then. Thanks, Rob!

**Review by davidzhangnyc — APPROVED** [review]

File: src/App.jsx (hunk 1 above)
Comment (Hydroelectric29) [reply src/App.jsx]: I commented in the slash page and push it to the upstream

**Review by JefZheng — APPROVED** [review]

File: src/components/about/About.jsx (hunk 2 above)
Comment (Hydroelectric29) [reply src/components/about/About.jsx]: Line 29 removed.

**Review by Hydroelectric29 — COMMENTED** [review]
Updated the splash page and removed line 29

**Pull request merged**